src/jquery.cycle2.carousel.js

```diff
@@ -143,7 +143,7 @@ $.fn.cycle.transitions.carousel = {
         var prepareDimensions = this.prepareDimensions;
 
         // throttle resize event
-        $(window).on( 'resize', resizeThrottle);
+        $(window).on( 'load resize', resizeThrottle);
 
         opts._carouselResizeThrottle = resizeThrottle;
         onResize();
```

Comment:
This should be removed to prevent onResize from being called twice. It will be called on window.load so we no longer need it here.